modules/cluster-logging-logstore-limits.adoc

```diff
@@ -39,15 +44,21 @@ spec:
         resources: <1>
           requests:
             cpu: "1"
-            memory: "16Gi"
+            memory: "64Gi"
+        proxy: <2>
+          limits:
+            memory: 100Mi
+          requests:
+             memory: 100Mi
 ----
-<1> Specify the CPU and memory requests as needed. If you leave these values blank,
-the Elasticsearch Operator sets default values that should be sufficient for most deployments.
-+
+<1> Specify the CPU and memory requests for Elasticsearch as needed. If you leave these values blank,
+the Elasticsearch Operator sets default values that should be sufficient for most deployments. The default values are`16Gi` for the memory request and `1` for the CPU request.
+<2> Specify the CPU and memory requests for the Elasticsearch proxy as needed. If you leave these values blank, the Elasticsearch Operator sets default values that should be sufficient for most deployments. The default values are`256Mi` for the memory request and `100m` for the CPU request.
```

Comment:
Not sure if it renders wrong or not, but there should be a space between "are" and "`256Mi`"

Reply:
Last-second change! Thanks for the catch.